entrypoint.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 
+target="${1:-versatilepb}"
```

Comment:
I think the underlying `$target` values should be: `pi1`, `pi2`, and `pi3`. It doesn't matter that they don't map 1:1 with the QEMU machine names.

Reply:
I like that, you could replace `versatilepb` with something else later if something better comes along.